File: src/main/java/com/ptah/entity/project/MileStone.java
package com.ptah.entity.project;

import com.ptah.common.BaseEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.ManyToOne;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Entity
@Getter
@Setter
public class MileStone extends BaseEntity {
    private String name;
    private Instant startTime;
    private Instant endTime;
    private String comments;
    @ManyToOne
    private Project project;
}

File: src/main/kotlin/com/ptah/entity/project/MileStone.kt
package com.ptah.entity.project

import com.ptah.common.BaseEntity
import jakarta.persistence.Entity
import jakarta.persistence.ManyToOne
import java.time.Instant

@Entity
class MileStone : BaseEntity() {
    var name: String? = null
    var startTime: Instant? = null
    var endTime: Instant? = null
    var comments: String? = null

    @ManyToOne
    var project: Project? = null
}
